pages/specification/migration/_index.md

```diff
@@ -3,5 +3,85 @@ title: Migrating from older drafts
 section: docs
 ---
 
-The release notes discuss the changes impacting users and implementers:
+### Introduction
 
+Just as your schemas evolve over time, the JSON Schema specification does as well. This guide provides useful information to make upgrading your schemas across versions of the specification a breeze.
+
+Learn how to use our migration guides and tooling ([AlterSchema](https://alterschema.sourcemeta.com/)) to bring your work with you.
+
+### Keywords Overview
+
+Here is a comprehensive overview to get you started.
+
+| All Keywords                | Specification | Draft introduction | Removed | Changed                                                                                      |
+| --------------------------- | ------------- | ------------------ | ------- | -------------------------------------------------------------------------------------------- |
+| `$anchor`                   | Core          | 2019-09            | No      | Partially replaced `$id`                                                                     |
```

Comment:
I think we should improve on the description of the change. Instead of just saying it partially replaces `$id`, we can mention something like the fact that `$id`s that defined anchor-only URI, like `$id: #foo` are now replaced with `$anchor`?

Reply:
My assumption is that this needs to be very brief to not explode the table. I'm not sure we have the space to explain the change properly. My understanding was that this kind of thing was going to be covered in detail elsewhere.